setup.cfg

```diff
@@ -26,7 +27,7 @@ packages = find:
 package_dir =
 include_package_data = True
 setup_requires = setuptools>=44; wheel; toml; setuptools_scm>=3.4.3
-python_requires = >=3.7
+python_requires = >=3.8
```

Comment:
I'd be reluctant to bump the python requirement unless there's an actual need.
AFAIK the codebase is still compatible with python 3.7.

Reply:
OK, nevermind. I figured I'd like to start using assignment expressions in ctypesgen, which require python 3.8.